manifold/src/face_op.cpp

```diff
@@ -50,12 +53,10 @@ void Manifold::Impl::Face2Tri(const VecDH<int>& faceEdge,
     ALWAYS_ASSERT(numEdge >= 3, topologyErr, "face has less than three edges.");
     const glm::vec3 normal = faceNormal[face];
 
-    std::map<int, int> vertBary;
-    for (int j = firstEdge; j < lastEdge; ++j)
-      vertBary[halfedge[j].startVert] = halfedgeBary.H()[j];
```

Comment:
Good call, I bet this could be a big part of why the triangulation step has been scaling worse than it should.